fineract-provider/src/main/java/org/apache/fineract/notification/service/NotificationDomainServiceImpl.java

```diff
@@ -247,6 +251,16 @@ public void onBusinessEvent(LoanCloseAsRescheduleBusinessEvent event) {
         }
     }
 
+    private class LoanChargebackTransactionListener implements BusinessEventListener<LoanChargebackTransactionBusinessEvent> {
+
+        @Override
+        public void onBusinessEvent(LoanChargebackTransactionBusinessEvent event) {
+            LoanTransaction loanTransaction = event.get();
```

Comment:
Please extract the strings ("LOAN_CHARGEBACK", "loanTransaction",...) into static final class variables.

Reply:
Done